scripts/vmbuild.rb

```diff
@@ -170,21 +170,24 @@ def verify_run(output)
   end
 end
 
-# Only update the symlink for a nightly
+# Only update the latest symlink for a nightly
 unless build_label == "test"
-  link = stream_directory.join("latest")
+  symlink_name = "latest"
+  link = stream_directory.join(symlink_name)
   if File.exist?(link)
     raise "#{link} is not a symlink!" unless File.symlink?(link)
     result = FileUtils.rm(link, :verbose => true)
     $log.info("Deleted symlink: #{result}")
   end
 
-  result = FileUtils.ln_s(destination_directory, link, :verbose => true)
-  $log.info("Created symlink: #{result}")
+  Dir.chdir(stream_directory) do
```

Comment:
if we're not in that directory, then we need to send the full path of the source file to ln -s, ending up in latest -> /build/.... path.   we really need latest -> directory_name as not all may have things mounted exactly the same way.